storybook/src/Heba/atoms/Toggles/Toggles.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Toggle, TOGGLE } from "./Toggle";
+
+export default {
```

Comment:
add disabled state with styling to the toggle.

Reply:
thanks,  will adding when my time allow isa

Reply:
please, comment back when you fix it

Reply:
thanks, will adding when my time allow isa